### PR TITLE
adding fastapi-mqtt extension

### PR DESCRIPTION
**Pull request by sabuhish**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +92,10 @@
 
 - [FastAPI Mail](https://github.com/sabuhish/fastapi-mail) - Lightweight mail system for sending emails and attachments (individual and bulk).
 
+### MQTT
+
+- [FastAPI MQTT](https://github.com/sabuhish/fastapi-mqtt) - fastapi-mqtt is extension for MQTT protocol.
```
Comment (Kludex) [inline README.md:97]:
```suggestion
- [FastAPI MQTT](https://github.com/sabuhish/fastapi-mqtt) - An extension for the MQTT protocol.
```

**Comment by mjhea0** [comment]
@Kludex thanks for the PR. This library looks great!

I'm hesitant to make "MQTT" an official category, though. 

1. Maybe a "Messaging" category so it includes AMQP as well?
2. Maybe an "Async Processing" category so it includes task queues and messaging protocols?
3. Or maybe just add this library to the "Utils" category for now? 

Open to suggestions.

**Comment by Kludex** [comment]
It's not my PR! :kissing: 

It's @sabuhish who suggested it. But since I'm here, I don't think we should create a category for a single resource. If we have another one, depending on what is it, we can define a name based on that group. What do you think?

**Comment by mjhea0** [comment]
Sounds good. Thanks!

**Comment by mjhea0** [comment]
Can you update the PR @sabuhish and move the resource to the "utils" category. Thanks!

**Comment by sabuhish** [comment]
Done ;)

**Pull request merged**